web/src/main/java/se/inera/intyg/minaintyg/auth/MinaIntygUser.java

```diff
@@ -17,9 +17,12 @@ public class MinaIntygUser implements Serializable {
   private final String personId;
   private final String personName;
 
-  public MinaIntygUser(String personId, String personName) {
+  private final LoginMethod loginMethod;
+
+  public MinaIntygUser(String personId, String personName, LoginMethod loginMethod) {
```

Comment:
Could we remove the hardcoded constructor? And the members shouldn't they have visibility private?

Because we serialise and deserialise this class you could add the constructor-annotations as well?